fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -56,6 +57,10 @@
         return idTranslator;
     }
 
+    protected Node asNode(final FedoraResource resource) {
```

Comment:
Javadoc added...
Please do not commit this PR without first squashing these two commits. I would be happy to do the squash if you prefer.